src/pages/About.vue

```diff
@@ -1,15 +1,138 @@
+                computing for all UC Berkeley students, faculty, and staff. We
+                run a super awesome computer lab on southside, have a wicked set
+                of servers for staff to play on, and provide cool services to
+                the Berkeley community.
+              </p>
+            </div>
+            <div class="column is-half">
+              <h2 class="subtitle">
+                Staff Meetings
+              </h2>
+              <p class="is-size-5">
+                OCF Staff meet to discuss technology, learn from each other, and
+                work on OCF projects. Drop-in visitors are always welcome,
+                especially those interested in joining the staff team!
+              </p>
+              <p class="is-size-4">
+                <strong>When:</strong> Every Monday at 8:10pm
+              </p>
+              <p class="is-size-4">
+                <strong>Where: </strong>
+                <a href="https://ocf.io/meet">Google Meet</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="is-divider"></div>
+    <section class="section">
+      <div class="container">
+        <div class="columns is-vcentered reverse-row-order">
+          <div class="column is-half">
+            <transition name="fade" mode="out-in">
+              <img
+                :src="servicesImages[servicesActiveImage]"
+                :key="servicesImages[servicesActiveImage]"
+                data-toggle="tooltip"
+                width="800"
+                height="600"
+                data-placement="bottom"
+                title="goTop"
+                alt=""
+              />
+            </transition>
+          </div>
+          <div class="column is-half">
+            <div class="columns is-centered is-vcentered">
+              <div class="column is-three-quarters">
+                <div class="content">
+                  <h2 class="title is-2 has-text-centered">
+                    Services We Provide
+                  </h2>
+
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 0"
+                    :class="{ 'active-card': servicesActiveImage == 0 }"
+                  >
+                    Web &amp; Email Hosting
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 1"
+                    :class="{ 'active-card': servicesActiveImage == 1 }"
+                  >
+                    UNIX Shell Accounts
+                  </p>
+                  <p
+                    class="button card box is-white is-medium"
+                    @click="servicesActiveImage = 2"
+                    :class="{ 'active-card': servicesActiveImage == 2 }"
+                  >
+                    Free Printing for Members
+                  </p>
```

Comment:
Something doesn't feel right about this section. I can't place my finger on it, but it's about how the "Services We Provide" meshes with the page.

Particularly, when the first button is active, it feels out of place. I would appreciate a second opinion on this though, I can't tell if it's just me, and this is definitely a nitpick, so I'm going to approve this otherwise.

Possible ways this might be resolvable:
1] the amount of shifting to the right could be reduced

----these are nonoptimal because the " Joining the OCF is an awesome way to learn! " blurb in the section below, which has the same formatting, would also have to be changed, but are possible considerations, and might help reviewers possibly see what I'm seeing.
2] the h2 could be increased in size
3] the distance between characters in h2 could be increased

For anyone just looking, for context, it's ![this I'm talking about](https://user-images.githubusercontent.com/69232394/97668185-b01e4380-1a3e-11eb-8386-82f831c0fa16.png)

Reply:
Yeah, the right shift is just one possible way to indicate which button is active, I'm open to other suggestions too.